lib/router/router.js

```diff
@@ -109,6 +126,17 @@ p.getInfo = function(req) {
     }
   }
 
+  // forward original image if no operation on image and browser support image format
+  if (routeInfo.imageSteps.length === 0 && (this.options.noop || routeInfo.urlInfo.query.noop)) {
```

Comment:
After thinking through this, it seems as `query.noop` has no value since you depend on `this.options.noop.imageTypeIfUnknown`. Please remove the query...

This is super hacky. You're basically saying that you don't want to even know the correct image format before responding to client with the untouched image. This is a big assumption that you're images are always the same format. 

I'm not a fan of this workflow but I can see your reasoning. Will merge once you remove the query param and merge latest from master.